lib/jobs/validate-ssh.js

```diff
@@ -123,6 +123,9 @@ function validationJobFactory(
             };
             conn.on('ready', function() {
                 conn.end();
+                if(sshSettings.host === null){
+                    reject('host ip is not defined in this lookup.');
```

Comment:
'lookup' may be some other words, 'lookup' in RackHD is easy to be understand as lookup documents.